api4/api.go

```diff
@@ -131,6 +132,7 @@ func Init(a *app.App, root *mux.Router) *API {
 	api.BaseRoutes.User = api.BaseRoutes.ApiRoot.PathPrefix("/users/{user_id:[A-Za-z0-9]+}").Subrouter()
 	api.BaseRoutes.UserByUsername = api.BaseRoutes.Users.PathPrefix("/username/{username:[A-Za-z0-9\\_\\-\\.]+}").Subrouter()
 	api.BaseRoutes.UserByEmail = api.BaseRoutes.Users.PathPrefix("/email/{email}").Subrouter()
+	api.BaseRoutes.UserAnalytics = api.BaseRoutes.Users.PathPrefix("/{user_id:[A-Za-z0-9]+}/useranalytics").Subrouter()
```

Comment:
Use user route here and add teamid as query param. ex
`api.BaseRoutes.User.PathPrefix("/teams/{team_id:[A-Za-z0-9]+}/useranalytics").Subrouter()`

Reply:
Completed.